README.md

```diff
@@ -65,33 +65,34 @@ This project supports below scenarios for end-to-end guest operating system vali
+| Ubuntu 18.04 live-server                               | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| Ubuntu 18.04 desktop                                   |                                  |                          | :heavy_check_mark:                 |
+| Ubuntu 20.04 and later                                 | :heavy_check_mark:               | :heavy_check_mark:       | :heavy_check_mark:                 |
+| Flatcar 2592.0.0 and later                             |                                  | :heavy_check_mark:       | :heavy_check_mark:                 |
+| Debian 10.10 and later, 11.x, 12.x                     | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| Debian 9.x, 10.9 and earlier                           |                                  |                          | :heavy_check_mark:                 |
+| Windows 10, 11                                         | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| Windows Server 2019, 2022                              | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| UnionTech OS Server 20 1050a                           | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| Fedora Server 36 and later                             | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| ProLinux Server 7.9, 8.5                               | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| FreeBSD 13 and later                                   | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| Pardus 21.2 Server,XFCE Desktop and later              | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| openSUSE Leap 15.3 and later                           | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| BCLinux 8.x                                            | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| BCLinux-for-Euler 21.10                                | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| Red Hat Enterprise Linux CoreOS (RHCOS) 4.13 and later |                                  | :heavy_check_mark:       | :heavy_check_mark:                 |
+| FusionOS 22 and 23                                     | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
```

Comment:
FusionOS or Fusion OS?

Reply:
FusionOS. Please check the official website: https://www.xfusion.com/cn/product/operating-system/fusionos

Reply:
Got it. Thanks!